addon/services/alexandria-documents.js

```diff
@@ -213,7 +214,31 @@ export default class AlexandriaDocumentsService extends Service {
         } catch (error) {
           document.category = previousCategory;
 
-          new ErrorHandler(this, error).notify();
+          if (error.errors?.[0]?.status !== "403") {
+            new ErrorHandler(this, error).notify();
+            return false;
+          }
+
+          const moveConfirmed = await confirm(
+            this.intl.t("alexandria.errors.move-failed", {
+              documentTitle: document.title,
+              categoryName: newCategory.name,
+            }),
+            {
+              confirmButtonText: this.intl.t("alexandria.confirm.copy"),
+              cancelButtonText: this.intl.t("alexandria.cancel"),
+            },
+          );
+
+          if (moveConfirmed) {
+            try {
+              await this.copy([id], newCategory);
+              return true;
+            } catch (copyError) {
+              new ErrorHandler(this, copyError).notify();
+              return false;
+            }
```

Comment:
Another question occurred to me: If multiple documents are moved, it will throw a permission error for all of them, and for each of them will ask to be copied, is that the intended behavior?

Reply:
reworked it. You will now be asked only one, whether to move one specific or to move all:
![image](https://github.com/user-attachments/assets/9d5e10c8-7a6b-456c-bcbb-1d8128a7a6c5)
![image](https://github.com/user-attachments/assets/26c5ddb4-3e5a-4414-8b30-06b3b22b3ccc)